packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -76,51 +77,38 @@ export class MarketOperationUtils {
         }
         const _opts = { ...DEFAULT_GET_MARKET_ORDERS_OPTS, ...opts };
         const [makerToken, takerToken] = getNativeOrderTokens(nativeOrders[0]);
+        const optionalSources = (this._liquidityProviderRegistry !== NULL_ADDRESS
+            ? [ERC20BridgeSource.LiquidityProvider]
+            : []
+        ).concat(this._multiBridge !== NULL_ADDRESS ? [ERC20BridgeSource.MultiBridge] : []);
```

Comment:
I wonder if we should just make `optionalSources` a `readonly` field so we only see this ugly bit of code once.